pkg/convert/pot.go

```diff
@@ -4,64 +4,42 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"regexp"
 	"strings"
 
-	"go.expect.digital/translate/pkg/model"
 	"go.expect.digital/translate/pkg/pot"
+
+	"go.expect.digital/translate/pkg/model"
 )
 
 type poTag string
 
 const (
-	MsgId  poTag = "msgid"
-	MsgStr poTag = "msgstr"
+	MsgId        poTag = "msgid"
+	PluralId     poTag = "msgid_plural"
+	MsgStrPlural poTag = "msgstr[%d]"
+	MsgStr       poTag = "msgstr"
 )
 
 func ToPot(m model.Messages) ([]byte, error) {
 	var b bytes.Buffer
 
-	if _, err := fmt.Fprintf(&b, "\"Language: %s\n", m.Language); err != nil {
+	if _, err := fmt.Fprintf(&b, "msgid \"\"\nmsgstr \"\"\n\"Language: %s\\n\"\n", m.Language); err != nil {
 		return nil, fmt.Errorf("write language: %w", err)
 	}
 
 	for i, message := range m.Messages {
-		if i > 0 {
-			if _, err := fmt.Fprint(&b, "\n"); err != nil {
-				return nil, fmt.Errorf("write new line: %w", err)
-			}
-		}
-
-		descriptions := strings.Split(message.Description, "\n")
-
-		for _, description := range descriptions {
-			if description != "" {
-				if _, err := fmt.Fprintf(&b, "#. %s\n", description); err != nil {
-					return nil, fmt.Errorf("write description: %w", err)
-				}
-			}
-		}
-
-		if message.Fuzzy {
-			if _, err := fmt.Fprint(&b, "#, fuzzy\n"); err != nil {
-				return nil, fmt.Errorf("write fuzzy: %w", err)
-			}
-		}
-
-		err := writeToPoTag(&b, MsgId, message.ID)
-		if err != nil {
-			return nil, fmt.Errorf("format msgid: %w", err)
-		}
-
-		err = writeToPoTag(&b, MsgStr, message.Message)
-		if err != nil {
-			return nil, fmt.Errorf("format msgstr: %w", err)
+		if err := writeMessage(&b, i, message); err != nil {
+			return nil, fmt.Errorf("write message: %w", err)
 		}
 	}
 
 	return b.Bytes(), nil
 }
 
 func FromPot(b []byte) (model.Messages, error) {
+	pluralCountLimit := 2
+
 	tokens, err := pot.Lex(bytes.NewReader(b))
 	if err != nil {
 		return model.Messages{}, fmt.Errorf("dividing po file to tokens: %w", err)
```

Comment:
try to be consistent with error messages, use Present Simple everywhere in errors
e.g

fmt.Errorf("divide ...")
fmt.Errorf("convert ...")